src/main/java/seedu/address/logic/LogicManager.java

```diff
@@ -39,9 +43,27 @@ public CommandResult execute(String commandText) throws CommandException, ParseE
         logger.info("----------------[USER COMMAND][" + commandText + "]");
 
         CommandResult commandResult;
+
+        /*
+        Step 10.
+        Modify parseCommand()
```

Comment:
Perhaps we can separate the two modes of Parsing into separate classes.
ie. `NormalModeParser` and `GameModeParser` each with their `parseCommand()` implementation